README.md

```diff
@@ -13,18 +14,29 @@ on your local machine for development purposes.
 ### Prerequisites
 
 #### Go
-A working setup for the Go Programming Language is needed. Here is a [getting started](https://golang.org/doc/install) guide. The project
-is currently using go version 1.11 along with module support.
+A working setup for the Go Programming Language is needed. Here is a [getting
+started](https://golang.org/doc/install) guide. The project is currently
+using go version 1.15 along with module support.
 
 #### Slack
 
-A slack account needs to be created as well as an [app](https://api.slack.com/apps). The app created is intended for development
-purposes. The following functionality must be enabled in the `Add features and functionality` section of the slack app configuration:
+NOTE: This project uses the original outdated Slack OAUTH 2.0 flow. Work is in
```

Comment:
For reference if it helps, https://github.com/jitsi/jitsi-slack/compare/jdm/slack_perms_migration. That branch should be most or all of what's needed. It didn't get push forward because there was still time left and wanted to avoid tracking an app submission at end of year.

Reply:
Thanks, I'll take a look at that! It also looks like slack-go/slack has added a bunch of V2 OAUTH2 features lately so I may just leverage that.